pubtools/_quay/push_docker.py

```diff
@@ -331,11 +332,13 @@ def generate_backup_mapping(self, push_items):
                 for tag in tags:
                     # repo doesn't exist, add to rollback tags
                     if not repo_data:
-                        rollback_tags.append(PushDocker.ImageData(full_repo, tag))
+                        rollback_tags.append(PushDocker.ImageData(full_repo, tag, None))
```

Comment:
Could you explain this None in the docstring? For a new reader it would be unclear why this is done, and that this structure is used for signature removal as well.

Reply:
I'll put here comments explaining why there's sometimes None and sometimes digest